articles/container-apps/tutorial-scaling.md

```diff
@@ -47,7 +47,7 @@ az containerapp up \
   --resource-group my-container-apps \
   --location centralus \
   --environment 'my-container-apps' \
-  --image mcr.microsoft.com/k8se/quickstart:latest \
+  --image mcr.microsoft.com/dotnet/samples:aspnetapp \
```

Comment:
Add a comment mentioning that you're using this image to help trigger scaling, and that you don't need to know or use .NET to complete this procedure.

Reply:
Added.